routers/repo/commit.go

```diff
@@ -195,7 +195,18 @@ func Diff(ctx *context.Context) {
 }
 
 func RawDiff(ctx *context.Context) {
-	panic("not implemented")
+	userName := ctx.Repo.Owner.Name
```

Comment:
Those variables are only been used once, I think we don't need to create them at all, just pass values to the `GetRawDiff` directly.